conf/prod.routes

```diff
@@ -88,7 +83,6 @@ GET         /personal-account/timeout
 GET         /personal-account/preferences                                              controllers.PaperlessPreferencesController.managePreferences
 GET         /personal-account/self-assessment-who-needs-to-register                    controllers.InterstitialController.displaySaRegistrationPage
 
-GET         /personal-account/tax-credits-are-ending                                   controllers.InterstitialController.displayTaxCreditsTransitionInformationInterstitialView
```

Comment:
Do not remove this url. it is still in use. Keep the redirect to tax-credits-have-ended

Reply:
fixed